src/API/WorkController.ts

```diff
@@ -62,6 +65,12 @@ export class WorkController {
     return { count, works }
   }
 
+  async getWorksCountByFilters(worksFilters: WorksFilters = {}): Promise<WorkCount> {
+    this.logger.trace({ method: 'getByFilters', worksFilters }, 'Getting Work by Filters from DB')
+    const count = await this.collection.find(worksFilters, { projection: { _id: false } }).count()
+    return { count }
```

Comment:
Can we just return an integer here instead of an object? It would better match the function name.

Reply:
Yeah that makes sense. I updated it.